.travis.yml

```diff
@@ -104,8 +104,11 @@ script:
     fi
     ~/.venv/bin/tox
 
+before_install:
+  - ~/.venv/bin/pip install codecov
```

Comment:
The venv is created in `install`, this is too late. Why don't you just install `codecov` in the same place `coveralls` used to be installed?